### DISCUSSION TITLE
FHIR mirth client - what is the use case?

### DISCUSSION THREAD

**Discussion by RamaKrishna-17** [nextgenhealthcare/connect#5647] [Q&A] [answered]
FHIR mirth client - what is the use case?

**Comment by pacmano1** [comment, marked as answer]
Maybe you can read the manual at https://docs.nextgen.com/bundle/Connect_FHIR_Connector_User_Guide/page/connect/connect/topics/c_overview_fhir_UG.html

You don't need the FHIR extension of course, FHIR is after all standard REST calls.   You can use https://hapi.fhir.org/ to help you construct proper calls.
